tsconfig.eslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["./packages/**/src/*", "./packages/**/test/*"],
+  "exclude": ["node_modules/"]
+}
```

Comment:
It will work only with ESLint configuration, not affect to compilation for typescript.